src/components/selectedbeast.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Modal, Button, Card} from "react-bootstrap";
+import "./select.css";
+
+class SelectedBeast extends React.Component {
+
+  render() {
+    return (
+      <div>
+
+      <Modal  show={this.props.show} onHide={this.props.handleClose}>
+        <Modal.Header>
+          <Modal.Title>{this.props.beastList.title}</Modal.Title>
+          <Card.Img
+            variant="top"
+            src={this.props.beastList.image_url}
+            alt="horned"
+            title="horned"
+          />
+        </Modal.Header>
+        <Modal.Body>
+            <Card.Text>{this.props.beastList.description}</Card.Text>
+        </Modal.Body>
+        <Modal.Footer>
+          <Button variant="secondary" onClick={this.props.handleClose}>
+            Close
+          </Button>
+        </Modal.Footer>
+      </Modal>
+      </div>
+    );
+  }
+}
```

Comment:
Get with a TA to help ya with the Modal 👍 

Currently, it isn't working as expected as it shows up on bottom of the page, but also, if I click, I am not actually always seeing the Modal. Check css and bootstrap code for Modal first. I know a TA can for sure help with this :)